scripts/constants.sh

```diff
@@ -6,9 +6,6 @@ GOPATH="$(go env GOPATH)"
 # Avalabs docker hub
 dockerhub_repo="avaplatform/avalanchego"
 
-# Current branch
```

Comment:
Can we keep this? This is very useful when running these builds

Reply:
Of course, I just searched everywhere in a repo and could not find where it is actually used.